git_machete/cmd.py

```diff
@@ -118,32 +118,33 @@ def right_arrow() -> str:
 
 
 class CommandLineContext:
-    opt_as_root: bool = False
-    opt_branch: Optional[str] = None
-    opt_checked_out_since: Optional[str] = None
-    opt_color: str = "auto"
-    opt_debug: bool = False
-    opt_down_fork_point: Optional[str] = None
-    opt_fetch: bool = False
-    opt_fork_point: Optional[str] = None
-    opt_inferred: bool = False
-    opt_list_commits: bool = False
-    opt_list_commits_with_hashes: bool = False
-    opt_merge: bool = False
-    opt_n: bool = False
-    opt_no_edit_merge: bool = False
-    opt_no_interactive_rebase: bool = False
-    opt_onto: Optional[str] = None
-    opt_override_to: Optional[str] = None
-    opt_override_to_inferred: bool = False
-    opt_override_to_parent: bool = False
-    opt_return_to: str = "stay"
-    opt_roots: List[str] = list()
-    opt_start_from: str = "here"
-    opt_stat: bool = False
-    opt_unset_override: bool = False
-    opt_verbose: bool = False
-    opt_yes: bool = False
+    def __init__(self) -> None:
```

Comment:
This could be a `@dataclass`... but AFAICS it's only supported since Python 3.7 without adding any extra dependency on some backport library :/

Reply:
Yeah, I thought about it but I share your concerns too